sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/util/Utilities.java

```diff
@@ -148,12 +149,21 @@ public static <T> T[] arraySlice(T[] data, int start) {
     }
 
     public static void runProcess(String directory, String... commands) throws IOException, InterruptedException {
-        ProcessBuilder processBuilder = new ProcessBuilder();
-        processBuilder.command(commands);
-        processBuilder.directory(new File(directory));
-        processBuilder.inheritIO();
+        File out = File.createTempFile("out", ".tmp", new File("."));
```

Comment:
What do these arguments do?

Reply:
well, it's a standard API. File name prefix, file name suffix, directory for the file.